docker/runtime/php-7.2/http-trigger/timeout.php

```diff
@@ -0,0 +1,17 @@
+<?php
```

Comment:
is this file needed? (it seems for debugging)

Reply:
Not needed and removed